packages/app/src/resource/ResourcePage.test.tsx

```diff
@@ -76,13 +76,8 @@ describe('ResourcePage', () => {
       fireEvent.click(screen.getByText('Delete'));
     });
 
-    try {
```

Comment:
This test passed when I added     
```
await act(async () => {
      await jest.runAllTimersAsync();
    });
```

However, I get a large, noisy warning about the mantine notifications not being wrapped in act(). There was no place to put it due to the fact that the warning will show before the mantine notification shows and then there would be an error if I put it above. I believe that this test is mainly to test the UI of the delete button rather than seeing if the delete button actually deletes (since our other tests solve that) so I removed it, but open to any other input.